week2-section1/exercise1/Main.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+* Exercise 1 - Understanding classes
+Create a class called FirstClass with a public constructor and destructor. Have the constructor output �I�m in the constructor!� and have the destructor output �I�m in the destructor!�. Then, instantiate 5 classes and run your program, are the constructor and destructor calls where you expected them to be?
+
+Now create a loop of at least five iterations. For each iteration, instantiate your class once. Run your program, are the constructor and destructor calls where you expected them to be?
+*/
+
+#include <iostream>
+
+class FMyFirtsClass
+{
+public:
+	FMyFirtsClass()
+	{
+		std::cout << "I'm in the constructor! " << std::endl;
+	};
+	~FMyFirtsClass()
+	{
+		std::cout << "I'm in the destructor! " << std::endl;
+	};
+};
+
+void InitializeClass() {
+	FMyFirtsClass firstClass;
```

Comment:
I want to get you used to being very explicit about your names. This code is fine, but naming your variables `firstClass`, `secondClass`, and so on is not entirely correct. Your class is `FMyFirtsClass`. It is your _only_ class. What you're doing here is creating _Instances_ of the class. So a more precise name could be `firstInstance`. What do you think?